Assets/RenderGraphs/RT_DEFERRED_PBR.lrg

```diff
@@ -512,53 +512,39 @@
             "x_dim_var": 1.0,
             "y_dim_var": 1.0,
             "z_dim_var": 1.0,
-            "draw_type": "CUBE",
-            "depth_test_enabled": true,
-            "cull_mode": "CULL_MODE_FRONT",
+            "draw_type": "FULLSCREEN_QUAD",
+            "depth_test_enabled": false,
+            "cull_mode": "CULL_MODE_BACK",
             "polygon_mode": "POLYGON_MODE_FILL",
             "primitive_topology": "PRIMITIVE_TOPOLOGY_TRIANGLE_LIST",
             "shaders": {
                 "task_shader": "",
                 "mesh_shader": "",
-                "vertex_shader": "/Helpers\\Cube.vert",
+                "vertex_shader": "/FullscreenQuad.vert",
```

Comment:
To avoid conflicts in the future, prefer to use the FullscreenQuad that is in the helpers folder

Change to /Helpers\\FullscreenQuad.vert
```suggestion
                "vertex_shader": "/Helpers\\FullscreenQuad.vert",
```